mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -206,21 +267,33 @@ export class ExperimentListView extends Component {
             aria-label='search experiments'
             value={searchInput}
             onChange={this.handleSearchInputChange}
+            onPressEnter={this.handleLoadMore}
             data-test-id='search-experiment-input'
+            style={{
+              width: '85%',
+            }}
           />
-          <div css={classNames.experimentListContainer}>
-            <Tree
-              treeData={treeData}
-              dangerouslySetAntdProps={{
-                selectable: true,
-                checkable: true,
-                multiple: true,
-                selectedKeys: activeExperimentIds,
-                checkedKeys: activeExperimentIds,
-                onCheck: this.handleCheck,
-                titleRender: this.renderListItem,
-              }}
-            />
+          <SearchIcon
+            onClick={this.handleLoadMore}
+            title='Search experiments'
+            css={{
+              fontSize: '24px',
+              marginLeft: 'auto',
+              width: '15%',
+            }}
+          />
+          <div id='scrollableDiv' css={classNames.experimentListContainer}>
+            <List split={false}>
+              <VirtualList
+                data={experiments}
+                height={this.containerHeight}
+                itemKey='experiment_id'
+                itemHeight={10}
+                onScroll={this.onScroll}
```

Comment:
Let's throttle or debounce `onScroll` for a few tens of milliseconds. Those functions should be already available from `lodash`.